tabs/tab_survival.py

```
@@ -268,244 +241,505 @@ def surv_ui(namespace: str) -> ui.TagChild:
+    # ==================== LABEL MAPPING LOGIC ====================
+    @reactive.Calc
+    def label_map():
+        """Build a dictionary mapping raw column names to user-friendly labels from var_meta."""
+        meta = var_meta.get() if hasattr(var_meta, 'get') else var_meta()
+        if not meta:
+            return {}
+        # Assuming meta is a list of dicts with 'name' and 'label' or similar structure
+        # Fallback to empty dict if structure is different
+        try:
+            return {item.get('name', k): item.get('label', k) for k, item in meta.items()}
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)